test/methods.jl

```diff
@@ -39,6 +39,16 @@ function test_equivalence(Λ, m1::RotationMethod, m2::RotationMethod; kwargs...)
     @test factor_correlation(r1) ≈ factor_correlation(r2) atol = 1e-5
 end
 
+@testset "factor rotation autodiff fallback" begin
+    method = ComponentLoss(abs2, orthogonal = true)
+    ∇Q = fill!(similar(A), NaN)
+
+    FactorRotations.set_autodiff_backend(:ABC)
+    @test_throws "ABC autodiff backend is not supported" criterion_and_gradient!(∇Q, method, A)
+    FactorRotations.set_autodiff_backend(:Enzyme)
+    #@test_throws "Enzyme.jl autodiff backend is not loaded" criterion_and_gradient!(∇Q, method, A)
```

Comment:
This test can be left in, no?

Reply:
Yes, I disabled it, because it requires that "using Enzyme" is delayed until after this command.
I've tweaked that,